### PR TITLE
Make sidebar sticky w/signoff

### LINKED ISSUE #252: Make the sidebar of the docs page sticky.
**Current Behavior**
The sidebar position is absolute

**Desired Behavior**
Make the sidebar of the docs page sticky.

---

**Resources**

**Alternatives / Additional Context**

---

**Contributor Resources**

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)

### PR DESCRIPTION
**Pull request by ritesh-karankal**
Signed-off-by: Ritesh Karankal <ritesh.karankal200@gmail.com>

**Description**

This PR fixes #252 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: docs/assets/css/sidebar_doc.css [hunk 1]
```diff
@@ -1,140 +1,136 @@
...
-    position: relative;
+  height: 100%;
+  margin-top: -30px;
+  position: fixed;
 }
 
 .sidebar {
-    position: absolute;
-    top: 30px;
+  position: absolute;
```
Comment (YashKamboj) [inline docs/assets/css/sidebar_doc.css:30]: Het @ritesh-karankal you have to change position of this class, also increase the top also

**Review by YashKamboj — CHANGES_REQUESTED** [review]

File: docs/assets/css/sidebar_doc.css (hunk 1 above)
Comment (ritesh-karankal) [reply docs/assets/css/sidebar_doc.css:30]:
```suggestion
.sidebar{
  position: fixed;
  top: 145px;

```
Is it correct?

Comment (YashKamboj) [reply docs/assets/css/sidebar_doc.css:30]: Yeah!! looks good @ritesh-karankal

**Review by YashKamboj — APPROVED** [review]
LGTM!! Good Work 👌

**Review by Abhijay007 — COMMENTED** [review]
@ritesh-karankal I think that formatting change can be ignored (if possible) as it is not required and sometimes it makes PR hard to review, else LGTM

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍 !!

**Review by debo19 — APPROVED** [review]
LGTM 🎉  Thanks @ritesh-karankal

**Pull request merged**